cli/lib/tasks/verify.js

```diff
@@ -65,7 +65,7 @@ const runSmokeTest = (binaryDir) => {
 
     debug('smoke test command:', smokeTestCommand)
 
-    return Promise.resolve(util.exec(cypressExecPath, args))
+    return Promise.resolve(util.exec(cypressExecPath, args, { stdio: ['pipe', 'pipe', process.stderr] }))
```

Comment:
One specific concern I have but am unsure how to test is, the `onSmokeTestError` callback contains:
```js
return throwFormErrorText(errors.missingDependency)(err.stderr || err.message)
```
And possibly the suggested change is hosing its ability to hook into `err.stderr`, for those cases where the smoke test exist with useful info on `stderr`.

Reply:
Just pushed an update that addresses this

Reply:
Welp. The previous approach passed CI, but the latest doesn't.

Anybody have thoughts on whether the previous approach https://github.com/cypress-io/cypress/pull/3807/commits/fb331f231ff88e5a4371d31d26a06008c5b252cd seems ok?